src/Umbraco.Web/Models/ContentEditing/ContentTypeCompositionDisplay.cs

```diff
@@ -22,14 +16,6 @@ protected ContentTypeCompositionDisplay()
 
         //name, alias, icon, thumb, desc, inherited from basic
 
-        //List view
-        [DataMember(Name = "isContainer")]
```

Comment:
because they already exist on the base class so they were basically "newing" them without the `new` keyword - now I realize that on the base class they are marked `[ReadOnly(true)]` - and then, I have no idea what that was all supposed to do - going to look into it

Reply:
well as per my tests, if the base class contains
~~~~
[Readonly(true)]
public int Value { get; set; }
~~~~
and the inheriting class contains
~~~~
public int Value { get; set; }
~~~~
which is not pretty but anyways - the model binder still treats the property as readonly

so no point to have these properties `new`ed at all